src/components/RegistrationForm.tsx

```diff
@@ -353,7 +353,11 @@ export default function RegistrationForm({ handles }) {
   const usernameValidator = useCallback(
     (value: string) => {
       const validationErrors: (string | React.ReactNode)[] = [];
-      if (handles.has(value)) {
+      if (
+        [...handles].find(
```

Comment:
Oh I thought `handles` was a list here 🙃

```suggestion
        Object.keys(handles).find(
```